CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## [Unreleased]
 ### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
+### Security
+
+## [0.3.0]
```

Comment:
@altavir 
The following issues are missing in the changelog:
https://github.com/mipt-npm/kmath/issues/286
https://github.com/mipt-npm/kmath/issues/176
https://github.com/mipt-npm/kmath/issues/364
https://github.com/mipt-npm/kmath/issues/254